plugin.rb

```diff
@@ -5,7 +5,6 @@
 # version: 1.0.1
 # authors: Sam Saffron
 # url: https://github.com/discourse/discourse-assign
-# transpile_js: true
```

Comment:
Personal question, but why are we changing this?

Reply:
This option is no longer used - all plugin js is now transpiled. (that line was removed from the plugin skeleton a year ago 😃 https://github.com/discourse/discourse-plugin-skeleton/pull/21)